woocommerce/woocommerce-gutenberg-products-block
# E2E tests for translated Cart & Checkout blocks

This e2e test checks weather the Cart & Checkout are translated properly. Some notable thoughts on this:

It doesn't check every translatable string as this would take too much config and setup for different scenarios. For example, to test that "Create an account" link is translated, there's a lot of setup of changing WooCommerce settings, block settings, logging in and out, etc. This is a different test in itself. I aim to check the basics. If I've missed something crucial or you feel strongly we should be testing every scenario, let me know.

I've opted for fetching the language files from the test directly rather than in `wp-env-config.sh` as we can clean these up at the end of the test and reset the env as it was for the remaining tests.


Fixes #5752 and #5769

### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. If E2E github actions pass in this PR then it's all good. 
2. You can alternatively run this specific test locally with `npm run test:e2e-dev -- cart-checkout-translations`
3. Check that the tests that run after this one pass and language is switched back to English successfully.
